app/scripts/actions.js

```diff
@@ -171,8 +170,15 @@ export function loadSuggestions (value) {
   return dispatch => {
     dispatch(loadSuggestionsBegin());
 
+    // If the value contains city, county or town strip it
+    // from the search
+    const normalizedValue = value.toLowerCase()
+      .replace('city', '')
+      .replace('county', '')
+      .replace('town', '');
```

Comment:
This may not do what is intended for a place like "Allentown, PA". But I guess the search should still work okay anyway.